src/mainwindow.cpp

```diff
@@ -2743,6 +2743,20 @@ bool MainWindow::eventFilter(QObject *object, QEvent *event)
             }
         }
         break;
+    case QEvent::KeyPress:
+    {
+        auto *keyEvent = static_cast<QKeyEvent *>(event);
+        if (keyEvent->key() == Qt::Key_Return && m_searchEdit->text().isEmpty()) {
+            setFocusOnText();
+            return true;
+        } else if (keyEvent->key() == Qt::Key_Return &&
+                   keyEvent->modifiers().testFlag(Qt::ControlModifier)) {
+            setFocusOnText();
+            return true;
+        }
+        return QMainWindow::eventFilter(object, event);
```

Comment:
Why is this line here? I guess it could just be removed because there is already `return QObject::eventFilter(object, event);` below (which could also use `QMainWindow`, but `QMainWindow` and `QWidget` don't override this virtual anyway).

Reply:
Ah, yes. I noticed it later. I will just convert into `break;` 
thanks